File: src/main/kotlin/org/jesperancinha/rockstarts/rockstarsmanager/repository/SongsRepository.java
package org.jesperancinha.rockstarts.rockstarsmanager.repository;

import org.jesperancinha.rockstarts.rockstarsmanager.model.Song;
import org.springframework.data.repository.CrudRepository;

public interface SongsRepository extends CrudRepository<Song, Long> {
    Song getSongsByName(String name);
}

File: src/main/kotlin/org/jesperancinha/rockstarts/rockstarsmanager/repository/SongsRepository.kt
package org.jesperancinha.rockstarts.rockstarsmanager.repository

import org.jesperancinha.rockstarts.rockstarsmanager.model.Song
import org.springframework.data.repository.CrudRepository

interface SongsRepository : CrudRepository<Song, Long?> {
    fun getSongsByName(name: String): Song
}